ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -289,6 +289,39 @@ def test_tensorflow_Asin(
     )
 
 
+# argmax
+@handle_cmd_line_args
+@given(
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("numeric", full=True),
+        valid_axis=True,
+        force_int_axis=True,
+        min_num_dims=1,
+    ),
+    output_type=helpers.get_dtypes("numeric"),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.tensorflow.ArgMax"
+    ),
+)
+def test_tensorflow_ArgMax(
+    dtype_x_axis, as_variable, num_positional_args, fw, native_array, output_type
+):
+    dtype, x, axis = dtype_x_axis
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="tensorflow",
+        fn_tree="raw_ops.ArgMax",
```

Comment:
the content of fn_tree argument should be the path to the origenal function in the target framework, in this case is math.argmax.

Reply:
I actually Implemented the one in rawops, There is another one with the same name in math

Reply:
Well, this is my mistake then, thank you.